app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -0,0 +1,13 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+export const calculateCreatedAgo = date => {
+  dayjs.extend(relativeTime);
```

Comment:
I want you to be careful here. I had explicitly pointed out how to handle this piece of code in the Slack channel, and thus I expect you to take care of it. If you are not taking care of it, then there are two possibilities:

- You don't agree with my suggestion or my suggestion doesn't work. Which I am fine with. We can discuss about it and rectify.
- You are still not attending to all Slack notifications. For this we will need to have a more serious talk.

Please do point out what exactly is the reason why you have not applied the change.

Reply:
I have forget to add `import dayjs from "lib/dayjs";` in `App.jsx`
That's why I need to write 
```
import dayjs from "dayjs";
import relativeTime from "dayjs/plugin/relativeTime";

export const calculateCreatedAgo = date => {
  dayjs.extend(relativeTime);
```
this block of code twice
to make it work

Reply:
https://www.loom.com/share/634a5a15c5774a2fb7c754438ab6d7aa

Reply:
Got it.

Reply:
@saurabh-nikam Please point out why you are importing the `dayjs` instance into `App.jsx`, like so:

```js
import dayjs from "lib/dayjs"
```

Are you not able to understand what's written in this section: https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=How%20to%20extend%20day.js%3F? If that's the case then please lemme know.

Reply:
After going through the above article thrice I as per my understanding , The correct way to extending dayjs globally is by adding this line `

> import "lib/dayjs";  

to root component or parent component where we want to use `dayjs`

Reply:
@saurabh-nikam Now you have the correct understanding.

But in order to further solidify your understanding, you can read the following:
- Singletons: https://rajeshnaroth.medium.com/singletons-in-javascript-and-es6-26f276a21df8#:~:text=The%20ES6%20Way%20%E2%80%94%20modules
- https://github.com/iamkun/dayjs/blob/dev/src/index.js#L395-L401